independent-projects/tools/devtools-testing/pom.xml

```diff
@@ -41,6 +41,13 @@
             <artifactId>assertj-core</artifactId>
             <scope>compile</scope>
         </dependency>
+
+        <dependency>
+            <groupId>io.quarkus</groupId>
+            <artifactId>quarkus-core-deployment</artifactId>
+            <version>${project.version}</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
Nope, this won't fly. We can't have a dependency to a Quarkus extension in an independent project.

Now, given the tests, I'm wondering if this project is really independent.

Reply:
Removed, and restore the initial test content.